src/main/cli/cli.c

```diff
@@ -5023,23 +5026,23 @@ const cliResourceValue_t resourceTable[] = {
     DEFS( OWNER_LED_STRIP,     PG_LED_STRIP_CONFIG, ledStripConfig_t, ioTag ),
 #endif
 #ifdef USE_UART
-    DEFA( OWNER_SERIAL_TX,     PG_SERIAL_PIN_CONFIG, serialPinConfig_t, ioTagTx[0], SERIAL_PORT_MAX_INDEX - SERIAL_LPUART_MAX_INDEX ),
-    DEFA( OWNER_SERIAL_RX,     PG_SERIAL_PIN_CONFIG, serialPinConfig_t, ioTagRx[0], SERIAL_PORT_MAX_INDEX - SERIAL_LPUART_MAX_INDEX ),
+    DEFA( OWNER_SERIAL_TX,     PG_SERIAL_PIN_CONFIG, serialPinConfig_t, ioTagTx[0], UART_PORT_COUNT ),
```

Comment:
Sadly, this won't work as expected.
On [src/main/target/STM32F411/target.h](https://github.com/betaflight/betaflight/pull/13569/files#diff-5f14a60f520daab84996c6afdee43233a20ebb1004fa761107d3acdf999e5d46), port 6 will have index 3, 

and `serialPortIdentifier_e` will result in wrong indexes.

It is possible to make it work (macro popcount when conversion ar compile time, function for SERIAL_PORT_IDENTIFIER_TO_INDEX for runtime code, but this will have to map almost 1to1 for user convenience (maximum port index may be used)

Reply:
hmm, yeah, I was wondering about that, i probably wrongly assumed that there was some pre-existing code going from identifiers to indexes.

I don't have time to investigate and fix this properly as I'm chasing down other bugs right now and dealing with hardware related activities.

The issue in #13567 still exists and should still be fixed prior to the release of 4.5.